backend/app/resources/create_user_info_dto.py

```diff
@@ -0,0 +1,28 @@
+import re
+
+emailRegex = r"^[a-z0-9]+[\._]?[a-z0-9]+[@]\w+[.]\w{2,3}$"
+phoneRegex = r"^[0-9]{9}$"
```

Comment:
I could be misreading this but isn't this requiring only 9 digits for a phone number? I think they're usually >= 10, e.g., `(416) 123-4567` (or `4161234567` in this case)